### ISSUE TITLE
Feature Request: Comment notification in Edit mode

### ISSUE THREAD

**Issue by mschwen** [mastbaum/philadelphia#31]
It would be nice to be able to pass hints to operators through the shift report.  On a busy shift (where you might need someone riding shotgun) you might not check email too often, and 'enthusiastic experts' might not want to call in to disturb you.  Maybe a full blown chat room would be appropriate.

**Comment by mastbaum** [comment]
Good call. Added to my evil plan for the next version, which should be a lot more collaborative. Got a lot of ideas from the "many chefs" situations in air fill...
